storage/store_pool.go

```diff
@@ -259,6 +261,18 @@ func (sp *StorePool) getStoreDescriptor(storeID proto.StoreID) *proto.StoreDescr
 	return &desc
 }
 
+// findDeadReplicas returns any replicas from the supplied slice that are
```

Comment:
The comments name this `findDeadReplicas`, but the method is actually called `deadReplicas`. I think `findDeadReplicas` is actually a better name as it contains a verb.